omni4_controller:
  type: "omni4_controller/Omni4Controller"
  rim_front_left_joint: "rim_front_left_joint"
  rim_front_right_joint: "rim_front_right_joint"
  rim_back_left_joint: "rim_back_left_joint"
  rim_back_right_joint: "rim_back_right_joint"
